client/src/features/signup/components/LoginInfoForm/LoginInfoForm.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Field } from 'formik';
+import { TextField } from 'formik-mui';
+import { Grid, Typography } from '@mui/material';
+import { formModel } from '../../data';
+
+const LoginInfoForm = () => {
+  const { email, login, password, retypePassword } = formModel;
+
+  return (
+    <>
+      <Typography variant="h3" mb={3}>
+        Login Info
+      </Typography>
+      <Grid container spacing={3} mb={3}>
+        <Grid item xs={12} laptop={6}>
+          <Field component={TextField} name={email.name} label={email.label} fullWidth required />
+        </Grid>
+
+        <Grid item xs={12} laptop={6}>
+          <Field component={TextField} name={login.name} label={login.label} fullWidth required />
```

Comment:
Here there is one mistake. Once I clicked on any field and do not type any data, I receive an error at each field even if I don't touch it. As far as I remember, we can write a custom input and receive props *touched* inside this component. Please, review the screen below. Here I receive props inside the Formik component and have access to the touched field. Guess it`s better to fix this issue here

![image](https://user-images.githubusercontent.com/112622259/202896668-71201d8c-94bc-4863-a7ac-6da3cdb1d580.png)
![image](https://user-images.githubusercontent.com/112622259/202896695-7c726252-e4dc-4823-8986-c5d38c6db871.png)

Reply:
am aware of it,  this functionality will be fixed in future updates as this branch is just for JSX  and styling